config/proguard/okhttp-proguard-rules.pro

```diff
@@ -10,3 +10,14 @@
 -dontwarn okhttp3.Authenticator*, okhttp3.CookieJar*, okhttp3.Dns*
 -dontwarn okhttp3.internal.http2.PushObserver*, okhttp3.internal.io.FileSystem*
 -dontwarn org.conscrypt.Conscrypt*
+
+# See: https://github.com/square/okhttp/issues/6258#issuecomment-894168816.
+-dontwarn org.bouncycastle.jsse.BCSSLSocket
+-dontwarn org.bouncycastle.jsse.BCSSLParameters
+-dontwarn org.bouncycastle.jsse.provider.BouncyCastleJsseProvider
+-dontwarn org.bouncycastle.provider.BouncyCastleJsseProvider
+-dontwarn org.bouncycastle.provider.BouncyCastleJsseProvider
+-dontwarn org.openjsse.javax.net.ssl.SSLSocket
+-dontwarn org.openjsse.javax.net.ssl.SSLParameters
+-dontwarn org.openjsse.net.ssl.OpenJSSE
+-dontwarn org.openjsse.net.ssl.OpenJSSE
```

Comment:
This appears to be duplicated--please remove the extra.

Reply:
Fixed